### PR TITLE
feat: Apply schema from collection model

### PR DESCRIPTION
**Pull request by JigarJoshi**
BREAKING CHANGE: Removed api to apply schema from dir and from JSON files. 

- added API to apply schemas from collection model.

### CODE REVIEW

File: client/src/main/java/com/tigrisdata/db/client/TigrisAsyncDatabase.java [hunk 1]
```diff
@@ -31,20 +29,13 @@ public interface TigrisAsyncDatabase {
...
-  CompletableFuture<ApplySchemasResponse> applySchemas(File schemaDirectory);
-
-  /**
-   * Creates or updates collection under current database.
-   *
-   * @param collectionsSchemas list of {@link URL}
-   * @return future to the {@link ApplySchemasResponse}
-   */
-  CompletableFuture<ApplySchemasResponse> applySchemas(List<URL> collectionsSchemas);
+  CompletableFuture<ApplySchemasResponse> applySchemas(
```
Comment (himank) [inline client/src/main/java/com/tigrisdata/db/client/TigrisAsyncDatabase.java:37]: we should call this CreateOrUpdateCollections now?

File: client/src/main/java/com/tigrisdata/db/client/TigrisDatabase.java [hunk 2]
```diff
@@ -30,24 +28,17 @@ public interface TigrisDatabase {
...
-   * current database.
+   * Applies the schema from collection models to current database
    *
-   * @param schemaDirectory directory containing schema files
+   * @param collectionModelTypes an array of collection model classes
    * @return response
    * @throws TigrisDBException in case of an error.
    */
-  TigrisDBResponse applySchemas(File schemaDirectory) throws TigrisDBException;
+  TigrisDBResponse applySchemas(Class<? extends TigrisCollectionType>... collectionModelTypes)
```
Comment (himank) [inline client/src/main/java/com/tigrisdata/db/client/TigrisDatabase.java:39]: ^

**Review by himank — APPROVED** [review]

File: client/src/main/java/com/tigrisdata/db/client/TigrisAsyncDatabase.java (hunk 1 above)
Comment (JigarJoshi) [reply client/src/main/java/com/tigrisdata/db/client/TigrisAsyncDatabase.java]: updated the method name.

File: client/src/main/java/com/tigrisdata/db/client/TigrisDatabase.java (hunk 2 above)
Comment (JigarJoshi) [reply client/src/main/java/com/tigrisdata/db/client/TigrisDatabase.java]: updated the method name.

**Review by himank — APPROVED** [review]

**Pull request merged**